Caboodle/Platform/Platform.android.cs

```diff
@@ -56,14 +57,17 @@ public static void BeginInvokeOnMainThread(Action action)
             handler.Post(action);
         }
 
-        internal static ClipboardManager ClipboardManager
-            => (ClipboardManager)Application.Context.GetSystemService(Context.ClipboardService);
+        internal static ClipboardManager ClipboardManager =>
+            Application.Context.GetSystemService(Context.ClipboardService) as ClipboardManager;
 
         internal static ConnectivityManager ConnectivityManager =>
-            (ConnectivityManager)Application.Context.GetSystemService(Context.ConnectivityService);
+            Application.Context.GetSystemService(Context.ConnectivityService) as ConnectivityManager;
 
         internal static WifiManager WifiManager =>
-            (WifiManager)Application.Context.GetSystemService(Context.WifiService);
+            Application.Context.GetSystemService(Context.WifiService) as WifiManager;
+
+        internal static SensorManager SensorManager =>
+            Application.Context.GetSystemService(Context.SensorService) as SensorManager;
```

Comment:
I am just thinking about `JavaCast` - is there a reason not to use that?

Reply:
Not sure... should we? I never have for these.

Reply:
Javacast may be faster, not sure... I know there's some sort of type registration done behind the scenes to allow the `as` to work with java.lang.object types.  My gut says JavaCast is probably the more explicit/safe way to go, but I don't have anything to back that up.